docs/pages/cloud/architecture.mdx

```diff
@@ -1,40 +1,37 @@
 ---
 title: Teleport Cloud Architecture
-description: Cloud security, availability and networking details.
+description: Cloud security, availability, and networking details.
 ---
 
 ## Security
 
-We have designed the Teleport Cloud environment to be secure; however we are still in the process of
-scrutinizing and executing on our security roadmap and working with independent security
-auditors to identify any gaps. Only once this is complete, the team will evaluate
-whether the Teleport Cloud is ready for strict compliance use-cases.
+We have designed the Teleport Cloud environment to be secure; and, security is an ongoing process of improvement.
+
+Our team routinely scrutinizes, reviews, updates, improves upon, and executes our security roadmap by working with independent security auditors to identify any gaps.
+
+Teleport Cloud only gets the green light for use in strict compliance use-cases only when the team has full confidence that all security gaps or vulnerabilities have been identified and resolved.
```

Comment:
Should we change the naming of Teleport Cloud etc, to better align with company messaging?

Reply:
Done.